build-system/tasks/presubmit-checks.js

```diff
@@ -566,6 +566,7 @@ var forbiddenTermsSrcInclusive = {
   // Functions
   '\\.changeHeight\\(': bannedTermsHelpString,
   '\\.changeSize\\(': bannedTermsHelpString,
+  '\\.attemptChangeHeight\\(0\\)': 'please consider using `attemptCollapse()`',
```

Comment:
That's what I was planning to do at first, just add a check inside `#attemptChangeSize`. Then we realize once an iframe send us request to resize to (0, 0), they will never be able to resize again because we don't remove the `display:none` for them.